### DISCUSSION TITLE
Terminal: "No module named tinytuya"

### DISCUSSION THREAD

**Discussion by cannondale0815** [make-all/tuya-local#1760] [General]
When trying to run `python -m tinytuya wizard` I get `/usr/bin/python: No module named tinytuya` .

My setup is a raspberry pi 4 with a supported Home Assistant OS installation:

Core 2024.3.1
Supervisor 2024.03.0
Operating System 12.1
Frontend 20240307.0

I am running Tuya Local 2024.3.1 and I believe to have followed the installation and setup instructions correctly (although am currently doubting myself a bit).

Also, I am using the "Advanced SSH & Web Terminal" Add-On version 17.2.0 and tried both the web Terminal as well as using SSH.

Thank you!

Correction: I incorrectly mentioned Local Tuya beforehand. I misspoke and meant Tuya Local.

**Comment by make-all** [comment]
Both command-line use of tinytuya and use of localtuya are out of scope for this discussion board.

**Reply by cannondale0815** [reply]
I misspoke and meant Tuya Local, not Local Tuya (which I was running before switching). I am using version 2024.3.1.

The instructions for Tuya Local make it seem that tinytuya is included and can be used on the command line without having to install it separately. Is that incorrect?

**Reply by make-all** [reply]
tinytuya is not included, but is listed as a dependency in the manifest, so if you install from HACS it should get installed into Home Assistant automatically.

The Advanced SSH & Web Terminal add on is a separate Docker container, so will not inherit Home Assistant's python environment.
